apps/xprof_gui/src/xprof_gui.app.src

```diff
@@ -0,0 +1,18 @@
+{application, xprof_gui,
+ [{description, "Visual BEAM profiler GUI"}
+ ,{vsn, "2.0.0-snapshot"}
+ ,{registered, []}
+ ,{mod, {'xprof_gui_app', []}}
+ ,{applications,
+   [kernel
+   ,stdlib
+   ,cowboy
+   ,lager
+   ,jsone
+   ]}
+ ,{env,[]}
+ ,{modules, []}
+ ,{licenses, ["BSD-3-Clause"]}
+ ,{maintainers, ["Michal Niec", "Pawel Pikula", "Peter Gomori", "Wojciech Gawronski"]}
```

Comment:
Maybe we should extend those lists? :wink:

Reply:
true, will do 👍